content/project/projects/contents+pt.Ir

```diff
@@ -0,0 +1,13 @@
+_model: page
+---
+_template: projects.html
+---
+title: Projetos
+---
+summary: O que compõe o projeto BeeWare?
+---
+sort_key: 4
+---
+body: BeeWare não é uma única ferramenta ou biblioteca - é uma coleção de projetos que podem ser usados ​​para ajudar a desenvolver, depurar e lançar software Python. Cada ferramenta segue a filosofia Unix de fazer algo bem. Cada ferramenta pode ser usada isoladamente, ou pode ser encadeada para fornecer um conjunto rico de ferramentas de programação.
```

Comment:
This file is already translated, the only reason why it's not showing the conflict here is that the name is incorrect (extension `.Ir` (capital i) instead of `.lr` (lowercase L) -- please remove it.

Reply:
Hi @eliasdorneles i will fixed it.